engine/installation/cloud/cloud-ex-aws.md

```diff
@@ -70,7 +70,7 @@ To get help with your private key file, instance IP address, and how to log into
         $ cd ~/Desktop/keys/amazon_ec2
         $ ssh -i "my-key-pair.pem" ubuntu@xx.xxx.xxx.xxx
 
-    We'll follow the instructions for installing Docker on Ubuntu at https://docs.docker.com/engine/installation/ubuntulinux/. The next few steps reflect those instructions.
+    We'll follow the instructions for installing Docker on Ubuntu [here](/engine/installation/linux/ubuntulinux.md). The next few steps reflect those instructions.
```

Comment:
That's great, but delete "here" and just have the text, "the instructions for installing Docker on Ubuntu" be the link text.